inline-gdocs-viewer.php

```diff
@@ -592,7 +592,8 @@ private function dataToHtml ($r, $options, $caption = '') {
         $id = ( 0 === $this->invocations )
             ? 'igsv-' . $this->getDocId( $options['key'] )
             : "igsv-{$this->invocations}-" . $this->getDocId( $options['key'] );
-        $html  = '<table id="' . esc_attr( $id ) . '"';
+        $id = esc_attr( $id );
+        $html  = '<table id="$id"';
```

Comment:
Unless I am misreading this, this hunk looks equivalent to the original. Why the change? If the functional execution of this hunk is the same as the original, please leave the original in place and remove this hunk from the commit. Thanks.